Sources/Network/TaxonomiesService.swift

```diff
@@ -20,7 +20,7 @@ enum TaxonomiesRoute: String {
     case getVitamins = "taxonomies/vitamins.json"
     case getMinerals = "taxonomies/minerals.json"
     case getNucleotides = "taxonomies/nucleotides.json"
-    case getInvalidBarcodes = "invalid-barcodes.json"
+    case getInvalidBarcodes = "taxonomies/invalid-barcodes.json"
```

Comment:
I made this change because the `refreshInvalidBarcodes()` request was always failing. But this url does not lead to the invalid-barcodes.json and I'm wondering if it has been removed entirely, I just can't find it.

Reply:
The right url is https://static.openfoodfacts.org/data/invalid-barcodes.json

Reply:
it's in a different directory than taxonomies